polyfill/test/regex.mjs

```diff
@@ -588,7 +588,7 @@ describe('fromString regex', () => {
     }
   });
 
-  describe('time zone ID', () => {
+  describe.skip('time zone ID', () => {
```

Comment:
Maybe worth a comment explaining that we can unskip these tests when we add the resolver parameter to from()?